preupg/creator/ui_helper.py

```diff
@@ -253,19 +260,37 @@ def _create_group_ini(self):
         config.set(section, 'group_title', 'Title for %s ' % self.get_group_name())
 
         file_name = 'group.ini'
-        group_ini = os.path.join(self.get_upgrade_path(), self.get_group_name(), file_name)
+        group_ini = os.path.join(self.get_upgrade_path(),
+                                 self.get_group_name(),
+                                 file_name)
         if os.path.exists(group_ini):
             return
-        try:
-            f = open(group_ini, 'wb')
-            config.write(f)
-            f.close()
-        except IOError:
-            print ('We have a problem with writing %s file to disc' % group_ini)
-            raise
+        UIHelper._write_to_file_content(group_ini, config)
+
+    def _create_properties_ini(self):
+        """
+        Create properties.ini inside module set directory in format:
+
+        [preupgrade-assistant-modules]
+        srcMajorVersion =
+        dstMajorVersion =
+        """
+        file_name = 'properties.ini'
+        section = 'preupgrade-assistant-modules'
+
+        config = ConfigParser.RawConfigParser()
+        config.add_section(section)
+        config.set(section, 'srcMajorVersion', '')
+        config.set(section, 'dstMajorVersion', '')
```

Comment:
@pirat89 I agree.

Just to remind, what the change would mean, p-c-c now works in 2 modes:

 *  found a new set + add new module,
 *  add new module to existing set.

In both cases it creates new module so it has to ask about that.  In second mode, it prepends the module question with for `y/n` confirmation.  Since the src/dst property is property of *the set*, we would want it to ask only in the first mode (set creation).

(The CLI could be improved but for now it's OK.)